src/plugins/intel_cpu/src/emitters/plugin/x64/jit_load_store_emitters.cpp

```diff
@@ -906,15 +910,15 @@ void jit_store_emitter::store_bytes(const Xbyak::Reg64& reg, int offset, int sto
         int bytes_to_store = store_size;
 
         if (store_size > 32) {
-            h->uni_vmovdqu(addr(0), ymm);  // store lower bits from zmm
+            h->uni_vmovdqu16(addr(0), ymm);  // store lower bits from zmm
             start_bytes += 32;
             bytes_to_store -= 32;
             // load upper bits from zmm into ymm
             STORE_KEEP_SOURCE(vextractf64x4, ymm, Ymm(aux_src_idx), zmm, 1);
         }
 
         if (bytes_to_store > 16) {
-            h->uni_vmovdqu(addr(start_bytes), xmm);  // store lower bits from ymm
+            h->uni_vmovdqu16(addr(start_bytes), xmm);  // store lower bits from ymm
             start_bytes += 16;
             bytes_to_store -= 16;
             // load upper bits from ymm into xmm
```

Comment:
Should we handle `vextractf128` below too? 
As far as I understand, if I change heuristic condition on L1022, the line below can be called on avx512 platform with 16-31 vector register but this instruction  `vextractf128` is [vex](https://www.felixcloutier.com/x86/vextractf128:vextractf32x4:vextractf64x2:vextractf32x8:vextractf64x4). What do you think - should we update this line too? Or at least to leave the comment too here that this is not executed on avx512.